server/lib/cache.py

```diff
@@ -53,7 +53,7 @@
       })
 
 cfg = lib_config.get_config()
-if cfg.USE_MEMCACHE:
+if cfg.USE_MEMCACHE or _redis_cache:
```

Comment:
where is _redis_cache set?

Reply:
Updated this check to create cache only if USE_MEMCACHE or REDIS_HOST is set -- this way we won't break any environments that had a redis cache configuration in gke, but had USE_MEMCACHE set to false (e.g., [autopush](https://github.com/datacommonsorg/website/blob/e2145ca5b6fffa5a14d7f0af798ae0233af6561d/server/app_env/autopush.py#L26))